presto-native-execution/presto_cpp/main/http/tests/certs/test_key1.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEowIBAAKCAQEAr5hRSoy8WPTzmvHFXFF8Ihkzz10oNSIDjBD/ANJqlxKK2OCp
+h6L6IauFDvpxYXkBHZTB+/LsN91KelocNz8V7+sK56T2VR3RZqWg3Hj5xjJFeCOj
+F6mAjILcRpl/LM2iPVCq8bqKzzu6fc3fQT+lEUHPWRALjpqq8V6nsWNATNIQyH9W
+QY2U0H3eRvzmEY7FBpgtUqQM47juzGgo7lJLqZsw0Inf8HMnlPjqcO+zmLKbs5wu
+zvDk4KqFZ9W33vnxXmBiu8C227lPnoYgVFI2SHmmQvUcs++cyMtBXM/2ox0YyJnJ
+OqM+Qt4sLzIkG8/4O6FrOTpnTNpT6BHhyIabgQIDAQABAoIBAF2c68G90+/s7y5s
+tJYqDu6y1u83L+dbsd5rdTd/9jmCNim3ZEoedhGbOn2owOeBNjoKA30JniuOO1WN
+UruVU2Grwg2K4ZSxJ+D8bgz8JdUnGegZv6ouwIyoezzcOnOtUVA1AF2M4NHoetz0
+84/6mUrg9eT6a97j0DyMBQTJ8A7Dj/dvcpibk7GCm/FaqYcvSSsb7z9iRzM9qZW7
+nOG7G25RXsVTQcX8cqlN9ztGJByOq+jD4gaRS5QGwFO2dyFvIYqes1sbMoCl3068
+Wm4woBmrWQrmQQZkFihFISA7FFA5jNyCtXhg0xifSfN5oPiXuq0RHFmA5U+bfT44
+0udufRECgYEA6XNUFSIJMRp1TwpXARqgjOzBrWc/G8J+/SWpYPqXzeFZPAC7qKi+
+IbLn3BmAK2w/Qc74oqFPPj231n29sprE9bRy5ID/TdSJvOI7KfLjVzXh3Ira8gY8
+owIy3UM/NZWTLv6E0THKy9mLJvE7X6gDFJ2AI4NxZQOj3GHDa0vI8SUCgYEAwI5c
+ZY9mZ6yjT5oIxDxZMur1aPCE9tVxkJ3CxRHsYUyDMDyEUIUg0jqpKWnLZy6e+ReZ
+nRX2dATT5HMnvt5q5y3dPen+Hidt8QA/FYRAN8d1QBX/CuG2jBoOQEo3MQexVuko
+uud/drOparsXVTf8BakjBVoOVOFUy2+qV9u52C0CgYBYMN9U1HTQfgRPBcL0gdCh
+LFfGYDixDfrXO0EPh9uNRVJmnohGIOki5ek9dsy6mrwS6lN3bKT5CGvItdvR+zHo
++HBm7myERIR8PKLJKB7GNW7L2BD6WWzSz+PTPrJEVesyV6hpNvw7okU1QvCBJIF1
+akKcJ+3QkV13xd8DS6/vmQKBgAoJYZeJmfTdcsW7aamQ27Vjudb2cnfLOFgpYPWU
+sZWAv3XMMblgtw0+MYMPGKMQMwkYydD+TBJQDhtiRTgVr8zM/zMy9yPrOjfNo3wj
+oJVHuDzW6KxQh4tV8Fg5hhbwJ7Y4AP99AY/WXYZKqjSu7lv9LO4ie4zBEgJol6Bs
+u4shAoGBAKbPrKUobbhsihDZsS8hjDQ9g0eMagxOJq57WqTEzAqih0k1YQQhjhHL
+fiANKrBbAFFOxdn1JfJNBdBXSEeVdJ+AMy7wj0K6dWGNP3A4FKqs5g5zb+grdPEJ
+l8qRFT+gODcZT3iSokEUuPnG0y07sM7CUS5rc94NqV/nE1dgQenA
+-----END RSA PRIVATE KEY-----
```

Comment:
Fix it? Thanks!

Reply:
I can do it, although originally such files does not have new line. Hence kept this way.